### ISSUE TITLE
createComment return error

### ISSUE THREAD

**Issue by FrancescoZ** [AnderGoig/SwiftInstagram#20]
## Checklist

- [x] I've the [latest update](https://github.com/AnderGoig/SwiftInstagram/releases) of Swiftinstagram.
- [x] I've searched for [existing GitHub issues](https://github.com/AnderGoig/SwiftInstagram/issues).
- [x] I've read the [wiki pages](https://github.com/AnderGoig/SwiftInstagram/wiki).
- [x] I've read the [contibution guidelines](CONTRIBUTING.md).

## Expected Behavior
Add a comment to a media

## Actual Behavior
Return error "Missing "text" attribute"

## Steps to Reproduce the Problem

  1. Login using the API with scope .comment .publicContent
  2. Call createComment

## Specifications

  - **iOS Version**: 11.2
  - **Device(s)**: iPhone X

**Comment by FrancescoZ** [comment]
I added this code every time I do a post request:

```
private func buildHTTPBody(withPatameters parameters: Parameters? = nil) -> Data? {
        var postString = "access_token=" + retrieveAccessToken()!
        parameters?.forEach({ (arg) in
            let (key, val) = arg
            postString.append("&" + key + "=" + String(describing: val))
        })
        return postString.data(using: .utf8)
    }
```

**Comment by AnderGoig** [comment]
Hi @FrancescoZ, I'm sorry for the late response. 

Huge thanks for letting me know that issue, I've also found more problems in the process of fixing it. I'm releasing a new version ASAP which should fix your problem.

Thanks again! 🤘

**Comment by AnderGoig** [comment]
Fixed in [v1.1.0](https://github.com/AnderGoig/SwiftInstagram/releases/tag/v1.1.0) 👈
